cypress/e2e/functional/view-transcript.cy.js

```diff
@@ -0,0 +1,40 @@
+import 'cypress-axe';
+import './commands';
+
+describe('Request Transcript', () => {
```

Comment:
I can see we already have the transcripts tab tested in the `case-file` and `request-transcript` files. Would it be better to extend the tests there instead of a adding a whole new journey which searches for a case etc.